src/lib/APS2.cpp

```
@@ -1083,11 +1128,11 @@ int APS2::get_PLL_freq() {
 			case 0x11: freq = 300; break;
 			case 0x00: freq = 600; break;
 			default:
-				return -2;
+				return -1;
```

should this throw an error rather than returning a magic number?